### PR TITLE
HDDS-2764. security/_index.md translation

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

doc

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2764

## How was this patch tested?

hugo server -D

### CODE REVIEW

**Comment by cxorm** [comment]
Thanks @iamabug for the translation.

Overall LGTM. Just a minor comment.

**Comment by cxorm** [comment]
LGTM.
Would you retest again ?

File: hadoop-hdds/docs/content/security/_index.zh.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+    -->
+
+{{<jumbotron title="安全化 Ozone">}}
+          Ozone 是一个企业级的安全存储系统，提供了很多可配置的安全特性，以下页面讨论了如何利用安全性配置来使 Ozone 安全化。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/security/_index.zh.md:26]: 提供了很多可配置的安全特性=>它提供了很多可配置的安全功能。

Comment (xiaoyuyao) [reply hadoop-hdds/docs/content/security/_index.zh.md:26]: 以下页面讨论了如何利用安全性配置来使 Ozone 安全化=》以下页面讨论了如何配置和使用 Ozone的 安全功能。

File: hadoop-hdds/docs/content/security/_index.zh.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{{</jumbotron>}}
+
+<div class="alert alert-warning" role="alert">
+如果你想要深入理解 Ozone 的安全架构，请查看 <a href="https://issues.apache.org/jira/secure/attachment/12911638/HadoopStorageLayerSecurity.pdf">Ozone 安全架构</a>。
+</div>
+
+根据需求不同，安全化 Ozone 有多种可选的步骤。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/security/_index.zh.md:33]: 根据需求不同-》根据不同的需求

**Comment by cxorm** [comment]
Thanks @adoroszlai for triggering CI check again and finally completed.
The work from @iamabug looks good to me, +1.

**Pull request merged**